currency/storage.go

```diff
@@ -571,6 +571,9 @@ func (s *Storage) GetExchangeRates() (Conversions, error) {
 func (s *Storage) SeedForeignExchangeRates() error {
 	s.fxRates.mtx.Lock()
 	defer s.fxRates.mtx.Unlock()
+	if s.fiatExchangeMarkets == nil {
```

Comment:
Same here, it is a problem to want to seed rates with no market to seed from

Reply:
Agreed.
Fixed in ae48ee8